services/thermal_mgr/thermal_mgr.h

```diff
@@ -5,7 +5,7 @@
 
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
-  
+  OS_INTERRUPT_EVENT
```

Comment:
its good style to keep ur prefixs for an enum consistent like naming this `THERMAL_MGR_EVENT_OS_INTERRUPT` or something